docs/topics/using-openrewrite-recipes.adoc

```diff
@@ -14,30 +14,44 @@ See link:{KBArticleTechnologyPreview}[Technology Preview features support scope]
 
 You can refactor the source code of Java applications by using link:https://docs.openrewrite.org/[OpenRewrite] recipes with the {ProductShortName} CLI.
 
-The OpenRewrite recipe `org.jboss.windup.JavaxToJakarta` renames imported `javax` packages to their `jakarta` equivalents.
+For example, the OpenRewrite recipe `org.jboss.windup.JavaxToJakarta` renames imported `javax` packages to their `jakarta` equivalents.
 
 .Procedure
 
-. Run`mta-cli`, specifying the recipe name and the application:
+. Run `mta-cli`, specifying the recipe name, the path to the configuration file, and the application:
 +
 [source,terminal]
 ----
-$ ./mta-cli --openrewrite "-DactiveRecipes=<recipe_name>"  \
-    --input </path/to/source/project> --goal dryRun
+$ ./mta-cli --openrewrite --input </path/to/source/project> \
+  "-DconfigLocation=<path/to/rewrite.yaml>  \
+  "-DactiveRecipes=<recipe_name>" --goal dryRun
 ----
 
-* `"-DactiveRecipes=<recipe name>"`: Specify the OpenRewrite recipe, for example, `org.jboss.windup.JavaxToJakarta`.
+* `--input`: Specify the application to be refactored.  The application must be the top of the source code project containing a Maven pom.xml
```

Comment:
Would it be more proper to write `containing a Maven Project Object Model (POM)`? The name of the file is `pom.xml`, but we're not supposed to refer to filenames as the thing itself.

Reply:
I suggest "Maven Project Object Model (POM) XML file" to avoid any ambiguity but still follow the style guides.